src/fe_values.jl

```diff
@@ -38,25 +38,33 @@ immutable FEValues{dim, T <: Real, FS <: FunctionSpace}
     detJdV::Vector{T}
     quad_rule::QuadratureRule{dim, T}
     function_space::FS
+    dMdξ::Vector{Vector{Vec{dim, T}}}
+    geom_space::FS
 end
 
-FEValues{dim, FS <: FunctionSpace}(quad_rule::QuadratureRule{dim}, func_space::FS) = FEValues(Float64, quad_rule, func_space)
+FEValues{dim, FS <: FunctionSpace}(quad_rule::QuadratureRule{dim}, func_space::FS, geom_space::FS=func_space) = FEValues(Float64, quad_rule, func_space, geom_space)
```

Comment:
Okay, is it enough to have like:
`FEValues{dim, FS1 <: FunctionsSpace, FS2 <: FunctionSpace}(...)`